.github/workflows/deploy_website.yml

```
@@ -0,0 +1,25 @@
+name: Deploy Website
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+on:
+  push:
+    branches:
+      - main
+jobs:
+  Deploy-Production:
+    runs-on: cloudquery-release-runner-8-cores
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Install Vercel CLI
+        run: npm install --global vercel@latest
+      - name: Pull Vercel Environment Information
+        run: vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
+      - name: Build Project Artifacts
+        run: vercel build --prod --token=${{ secrets.VERCEL_TOKEN }}
+      - name: Deploy Project Artifacts to Vercel
+        run: vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
```

```suggestion
        run: vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
```

Is `-prod` is the only difference from the preview workflow?
We can do a conditional step based on the branch name https://docs.github.com/en/actions/learn-github-actions/expressions#example-expression-in-an-if-conditional.

Also I'd also run on pushes to main and pull requests to main (not on every push to every branch), as that would increase our build usage significantly

---

Updated to run on pushes to main and pull request 👍  For the two file setup, I was mostly just following the recommendation here: https://vercel.com/guides/how-can-i-use-github-actions-with-vercel

All three steps have some small difference so I think it's probably okay to keep them separate, but if you want we can consolidate in a follow-up also